### ISSUE TITLE
SDR Output

### ISSUE THREAD

**Issue by Hibby** [F5OEO/rpidatv#41]
I'd like to be able to output generated I/Q to an SDR - in my case the hackRF. 

This would allow a fairly cheap set up with kit that a PA can be easily attached to. 

Something like [SoapySDR](https://github.com/pothosware/SoapySDR/) as a layer to abstract away from a specific vendor/model might be a solid starting point.

**Comment by F5OEO** [comment]
Yes..that is planned. Output I/Q is trivial. Sending to SoapySDR ang go inside a SDR has to be written. Contributors welcomed !

**Comment by F5OEO** [comment]
Really need to be done ! Have the target to be ready when LimeSDR will arrive
